### PR TITLE
Upgrade 'The Hit List' to 1.1.17

### PR DESCRIPTION
**Pull request by hutschen**

### CODE REVIEW

File: Casks/the-hit-list.rb [hunk 1]
```diff
@@ -1,11 +1,10 @@
 cask :v1 => 'the-hit-list' do
-  version :latest
-  sha256 :no_check
+  version '1.1.17'
+  sha256 '0a6bd16d26c19a27f2146cdba27fc5c8119da8c382643d285203b41fadaebe98'
 
-  url 'http://www.potionfactory.com/files/thehitlist/TheHitList.zip'
-  appcast 'http://www.potionfactory.com/appcast/thehitlist.php'
+  url 'http://distrib.karelia.com/downloads/TheHitList-302.zip'
```
Comment (vitorgalvao) [inline Casks/the-hit-list.rb]: What does the `302` mean? Is it the build number?

File: Casks/the-hit-list.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 cask :v1 => 'the-hit-list' do
-  version :latest
-  sha256 :no_check
+  version '1.1.17'
+  sha256 '0a6bd16d26c19a27f2146cdba27fc5c8119da8c382643d285203b41fadaebe98'
 
-  url 'http://www.potionfactory.com/files/thehitlist/TheHitList.zip'
-  appcast 'http://www.potionfactory.com/appcast/thehitlist.php'
```
Comment (vitorgalvao) [inline Casks/the-hit-list.rb]: Don’t remove the appcast. Why would you? It’s still working.

File: Casks/the-hit-list.rb (hunk 1 above)
Comment (hutschen) [inline Casks/the-hit-list.rb]: I'm not pretty sure but I think it's the build number. I was not able to find a generic download link.

File: Casks/the-hit-list.rb (hunk 2 above)
Comment (hutschen) [inline Casks/the-hit-list.rb]: Sorry, the download url wasn't working so I supposed the same for the appcasts' url and don't checked it before I removed it. I just found out that the appcasts' url redirects to another one which I added to the file now.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as f1e00110d58101668f6383a42ed25314318b7142 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**